crates/bevy_math/src/bounding/mod.rs

```diff
@@ -43,6 +43,9 @@ pub trait BoundingVolume {
 
     /// Decrease the size of the bounding volume in each direction by the given amount
     fn shrink(&self, amount: Self::HalfSize) -> Self;
+
+    /// Scale the size of the bounding volume around the center by the given amount
```

Comment:
"the center" could be confused with the global origin, i.e. (0, 0, 0). It should explicitly refer to the center of the bounding volume.

```suggestion
    /// Scale the size of the bounding volume around its center by the given amount
```